docs/docs/troubleshooting.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Troubleshooting
+description: >-
+  Learn how to troubleshoot common configuration issues or work around any outstanding bugs.
+sidebarDepth: 0
+lang: en-US
+meta:
+  - name: keywords
+    content: pomerium troubleshooting
+---
+
+# Troubleshooting
+
+This article provides troubleshooting information for various tools and features from Pomerium.
+
+[[toc]]
+
+## Pomerium Core
+
+### JWT Authentication with Let's Encrypt
+
+Wildcard certificates signed by LetsEncrypt may still be cross-signed by the [expired DST R3 root](https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/). While many browsers still trust these certificates (as long as they are also signed by a valid root), some upstream applications reject them, including Grafana as an example:
```

Comment:
- Can we provide some additional context here around the different certificates at play? (JWKS / signing keys VS the lets encrypt certificates )
-  Maybe a little more explanation on why the LetsEncrypt issue results in the upstream client not being able to pull down and cache the jwks keys

Reply:
Adjusted in https://github.com/pomerium/pomerium/pull/2797/commits/414d1cd70d80d03ec22f6947f31da44bb36530db